### ISSUE TITLE
Prune history

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-page-template#4]
The way this is currently done, the website draws in over 50 MB of history. This is more than DKPro Core has! May there be a way to purge some of the older history of the template? I don't think we have to track this down into prehistoric ages.

**Comment by reckart** [comment]
Looks like it is possible to prune history:

http://stackoverflow.com/questions/4515580/how-do-i-remove-the-old-history-from-a-git-repository

However, if we do it, we can no longer merge in changes from the original template repo from
which we initially forked. Well... we can, but not with git I guess - rather with regular diffs/patches.

Anyway, I'd prefer not imposing a 50 MB overhead on every DKPro repository just for website template archeology.

So I suggest we do one of the two following things:

1) prune the history of the dkpro-page-template to exclude everything before 33cc7c4 (one change before Erik-Lân started making changes) and loosing the ability to merge in changes from the original repo

2) change the instructions for creating a gh-pages based on our template in such a way that it does not include the full history (or maybe no history at all) and loose the ability to merge in changes from dkpro-page-template

Any preferences? I personally lean towards 1.

**Comment by reckart** [comment]
Out-of-band votes from JD and JEK: +1 for option 1

**Comment by dodinh** [comment]
+1 for 1

**Comment by reckart** [comment]
I did it a bit differently than proposed: instead of truncating the history up to a certain commit, I deleted the history of all files that are no longer in the repository. It brought the size of the repo down to < 3 MB.

The procedure was performed using the accepted answer from this post: http://stackoverflow.com/questions/17901588/new-repo-with-copied-history-of-only-currently-tracked-files

Subsequently, I have done a little renaming of branches until I had replaced the contents of this repo with the pruned gh-pages - thus the issues are still here.

To anybody who had already tried to set up a gh-pages branch based on this template: **delete the gh-pages branch from your project and repeat the procedure**.
